### ISSUE TITLE
raster_to_vector_polygons missing

### ISSUE THREAD

**Issue by mlavy** [jblindsay/whitebox-tools#70]
the raster_to_vector_polygons tool from WhiteboxTools is missing in the whitebox python package.

**Comment by jblindsay** [comment]
This tool is not being left out of the Python package. It simply does not exist yet. I have started to write the tool, which is why the file exists in the repository, but it is not yet connected to the larger program because it is far from complete. Once the tool is complete, I will add it to WBT, including the Python package. You're about the third person now to make this request, and so I will bump it to a higher priority but I'm not sure if it will make the next release or not, as I don't want to delay the release and this is quite a complex tool.

**Comment by jblindsay** [comment]
I've just added the code for the RasterToVectorPolygons tool, which has now passed testing and will be included in the next version of WhiteboxTools, to be release later this week. I'm marking this as closed but if you encounter any issues with the tool during your own testing, feel free to open another issue.
